README.md

```diff
@@ -4,28 +4,33 @@
 
 ## What is Rafiki?
 
-Rafiki is an open source package that exposes a comprehensive set of
-Interledger APIs. It's intended to be run by wallet providers, allowing them to
-offer Interledger functionality to their users.
+Rafiki is an open source package that allows an [Account Servicing Entity](./docs/glossary.md#account-servicing-entity) to enable [Interledger](./docs/glossary.md#interledger-protocol) functionality on its users' accounts.
```

Comment:
"package" might be confusing since we refer to individual monorepo packages below in 
https://github.com/interledger/rafiki#local-development

Reply:
project?